src/lib/objects/payggy/views/selectUser.svelte

```diff
@@ -0,0 +1,108 @@
+<script lang="ts">
```

Comment:
In other files the naming convention is to use dashes as separator between words. According to that this file should be called `select-user.svelte` and not `selectUser.svelte`